client/rest-high-level/src/main/java/org/elasticsearch/client/ml/dataframe/DataFrameAnalyticsConfig.java

```diff
@@ -56,6 +56,7 @@ public static Builder builder() {
     private static final ParseField MODEL_MEMORY_LIMIT = new ParseField("model_memory_limit");
     private static final ParseField CREATE_TIME = new ParseField("create_time");
     private static final ParseField VERSION = new ParseField("version");
+    private static final ParseField ALLOW_LAZY_START = new ParseField("allow_lazy_start");
```

Comment:
For the equivalent setting in anomaly detection jobs, it can be added to the modules the SIEM team is using to configure their ML jobs.  Then every time those jobs are started they'll pick up the setting.  Basically with the work flows that are envisaged for this new setting we think that it will be something that's always desired or not desired for a particular job, and not desired for some invocations but not others.

Then for the data frame analytics setting specifically I made it symmetrical to the way things are done for anomaly detection jobs.